opencga-core/src/main/java/org/opencb/opencga/core/config/QuotaConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package org.opencb.opencga.core.config;
+
+public class QuotaConfiguration {
+
+    private int maxNumUsers;
+    private int maxNumProjects;
+    private int maxNumIndexedSamples;
+    private int maxNumJobHours;
+
+    public QuotaConfiguration() {
+    }
+
+    public QuotaConfiguration(int maxNumUsers, int maxNumProjects, int maxNumIndexedSamples, int maxNumJobHours) {
+        this.maxNumUsers = maxNumUsers;
+        this.maxNumProjects = maxNumProjects;
+        this.maxNumIndexedSamples = maxNumIndexedSamples;
+        this.maxNumJobHours = maxNumJobHours;
+    }
+
+    public static QuotaConfiguration init() {
+        return new QuotaConfiguration(15, 5, 15000, 100);
```

Comment:
So, by default, you're now applying a quota of 100h max everywhere?

Reply:
In case of missing values in the configuration file (as if you were upgrading from a previous version), I would not use any quota at all.

Reply:
Maybe you're right. Setting default quota values only in the configuration file